packages/openapi-ts/src/plugins/@tanstack/svelte-query/types.d.ts

```diff
@@ -104,7 +126,7 @@ export type UserConfig = Plugin.Name<'@tanstack/svelte-query'> & {
   /**
    * Configuration for generated mutation options helpers.
    *
-   * See {@link https://tanstack.com/query/v5/docs/framework/solid/reference/createMutation}
+   * See {@link https://tanstack.com/query/v5/docs/framework/svelte/reference/createMutation}
```

Comment:
Huh, thanks!

Reply:
Actually not yet thanks haha. I think I left it referencing Solid because there were no proper docs. You can see your link throws 404. But https://tanstack.com/query/v5/docs/framework/svelte/reference/functions/createmutation works so we could use that.